.github/workflows/playwright.yml

```diff
@@ -15,6 +15,11 @@ on:
         description: Plugin version
         type: string
         required: true
+      plugin-directory:
+        description: Directory of the plugin, if not in the root of the repository.
+        type: string
+        required: false
+        default: .
```

Comment:
This input also be added to `ci.yml` and `cd.yml`, and passed down to the `playwright.yml` workflow

Reply:
Done in a58e9b9a36cd0008c75c4b8deda17668e2a1ba87.